addons/account/models/account.py

```diff
@@ -1460,7 +1460,7 @@ def _compute_amount(self, base_amount, price_unit, quantity=1.0, product=None, p
             return base_amount / (1 - self.amount / 100) - base_amount if (1 - self.amount / 100) else 0.0
         # <=> new_base * (1 - tax_amount) = base
         if self.amount_type == 'division' and price_include:
-            return base_amount - (base_amount * (self.amount / 100))
+            return base_amount * (self.amount / 100)
```

Comment:
This is completely wrong!
You computation needs to be symmetrical like percentage. Example with percent:
price_exclude:
tax = base * percent <=> base + tax = base * (1 + percent) = base_with_tax
price_include:
base_with_tax / (1 + percent) = base <=> tax = base_with_tax - base

Your fix is wrong because you set the same formula as `if self.amount_type == 'percent' and not price_include:` but this is not the opposite computation of `return base_amount / (1 - self.amount / 100) - base_amount if (1 - self.amount / 100) else 0.0`